src/process.h

```diff
@@ -21,11 +21,13 @@ struct ProcessInfo {
   DWORD pid;
   DWORD ppid;
   DWORD memory; // Reported in bytes
+  TCHAR arguments[512];
```

Comment:
I notice below you trim if the command line goes over, is there a actual maximum value we could use here?

Reply:
The actual maximum is [8191](https://stackoverflow.com/questions/3205027/maximum-length-of-command-line-string) (at least for processes run from cmd line), but increasing it beyond ~720 causes the JS tests to error out without any message. I tried switching to allocating the ProcessInfo array on the heap, but that causes a heap out of memory exception when running the tests with the current size.

Earlier I thought I was seeing nondeterministic failures when it was set to 720, so I'm wary of increasing it close to that. My guess is that it failed because it wasn't able to find a piece of contiguous memory large enough, even though its not a particularly large array, ~54k.